src/pages/AdminPage/AdminUserModal.tsx

```diff
@@ -43,11 +45,25 @@ export const AdminUserModal = ({
   open: boolean;
   onClose: () => void;
 }) => {
+  const { account } = useWeb3React();
   const classes = useStyles();
 
   const { circle: selectedCircle, circleId } = useSelectedCircle();
   const { updateUser, createUser } = useApiAdminCircle(circleId);
 
+  const [showOptOutChangeWarning, setShowOptOutChangeWarning] = useState(false);
+  const [hasAcceptedOptOutWarning, setHasAcceptedOptOutWarning] =
+    useState(false);
+
+  useEffect(() => {
```

Comment:
If you apply [this change](https://github.com/coordinape/coordinape/commit/ffe6f35cf1e0ed42ec92fd5431abd9c981736879) to your PR then this `useEffect` becomes unnecessary. I'd do it myself but I can't push changes back to your fork